### ISSUE TITLE
Blackhole community RFC7999

### ISSUE THREAD

**Issue by mrimann** [NLNOG/bgpfilterguide#68]
It would be awesome to have some examples on how to implement [RFC7999](https://www.rfc-editor.org/rfc/rfc7999) (BLACKHOLE BGP Community) with the several platforms

For both
- the announcing part (e.g. how to announce to blackhole an IP address)
- and for filtering (e.g. how to check received announcements for certain communities and thus deny to accept them).

**Comment by mrimann** [comment]
@job What about this one? Is there something important missing?

**Comment by teunvink** [comment]
At this moment the BGP Filter Guide is intended a a guide for building import filters, consisting of generic building blocks which can be applied by anyone. The 'announce part' as you call it is part of an export filter, and should, just as the import filter you're suggesting, be part of a well thought out set of rules for managing your network. 

Therefor, these suggestions are not really suited as generic snippets in a guide like this.

**Comment by mrimann** [comment]
@teunvink thanks for your feedback. I agree, this must be customized and cannot be copy-pasted 1:1 like other things that are more generalized.

When I started configuring routers and BGP the first time, the BGP Filter Guide was of much value to me and I learned a lot about best practices - and how to implement a config per product based on those best practices.
Since stuff like the whole blackholing configuration was not part of the Guide, I thought I'd propose them as an addition, get feedback from the professionals, fine-tune those sections and by adding that topic, help others that are not (yet) very well versed with BGP configs (as in learn from working examples).

I'm fine with not having my proposal accepted into the guide and this is not about complaining. Just trying to give a different perspective why I thought this could be a beneficial addition.
